app/views/practices/show/desktop_partials/_maturity_level_banner.html.erb

```diff
@@ -6,7 +6,7 @@
     %>
     <p class="usa-alert__text margin-bottom-0">
       <%= maturity_map[:description] %>
-      <a href="/search?query=<%= maturity %>" class="usa-link">See more <%= maturity_map[:link_text] %> practices.</a>
+      <a href="/search#filter_by=<%= maturity %>" class="usa-link">See more <%= maturity_map[:link_text] %> practices.</a>
```

Comment:
We don't want to pass query params, because we are doing a filtered search. We want to separate the two use cases in which, A. the user actually types in replicate in the search input and B. The user clicks on a maturity level link. Hope this makes sense.

Reply:
I guess I am asking does `/search?filter_by` not work since I have only seen the `#` symbol in urls help to navigate to a certain section of the page if that makes sense 😅

Reply:
Okay, I understand. Yes, I believe it would work. If you think this is the better way to go, I will make that change. Thanks Elisa!